app/workers/mails.php

```diff
@@ -76,6 +76,11 @@ public function run(): void
         /** @var \PHPMailer\PHPMailer\PHPMailer $mail */
         $mail = $register->get('smtp');
 
+        // https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting#enabling-debug-output
+        if (App::getEnv('_APP_ENV') !== App::MODE_TYPE_PRODUCTION) {
```

Comment:
This method makes more sense to me: https://github.com/utopia-php/framework/blob/master/src/App.php#L411
I would just double check we initialize the env setting in the worker scope the same way we do for the REST API.